code/datums/tutorial/marine/medical_basic_item.dm

```diff
@@ -0,0 +1,325 @@
+/datum/tutorial/marine/medical_basic_item
+	name = "Marine - Medical Tutorial 2 (Standard Tools)"
+	desc = "Learn the medical tools you'll need to use to heal yourself and others"
+	tutorial_id = "marine_medical_2"
+	tutorial_template = /datum/map_template/tutorial/s8x9
+
+	var/clothing_items_to_vend = 7
+	var/med_one_items_to_vend = 1
+	var/med_two_items_to_vend = 1
+	var/med_three_items_to_vend = 1
+
+// START OF SCRIPTING
+
+/datum/tutorial/marine/medical_basic_item/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	init_mob()
+	message_to_player("This tutorial will discuss the medical tools you will use as a standard Marine.")
+	addtimer(CALLBACK(src, PROC_REF(intro_one)), 6 SECONDS)
+//pouch_one intro_one
+/datum/tutorial/marine/medical_basic_item/proc/intro_one()
+	message_to_player("You are capable of tending to your own basic injuries without needing Corpsman (sometimes called a Medic) to assist you.")
+	update_objective("")
+	addtimer(CALLBACK(src, PROC_REF(intro_two)), 6 SECONDS)
+
+/datum/tutorial/marine/medical_basic_item/proc/intro_two()
+	message_to_player("The knowledge and ability to tend to your own injuries will allow you to not only survive, but thrive, on the battlefield.")
+	update_objective("")
+	addtimer(CALLBACK(src, PROC_REF(kit_one)), 6 SECONDS)
+
+/datum/tutorial/marine/medical_basic_item/proc/kit_one()
+	message_to_player("We will now show and discuss the basic medical items you can use. Afterwards, we will examine where you can find and store these items.")
+	update_objective("")
+	addtimer(CALLBACK(src, PROC_REF(kit_two)), 6 SECONDS)
+
+/datum/tutorial/marine/medical_basic_item/proc/kit_two()
+	SIGNAL_HANDLER
```

Comment:
`SIGNAL_HANDLER` is only needed if you're calling this from a signal using `RegisterSignal()`. Not needed if you're just using a timer to call it.